modules/nf-core/dada2/tests/main.nf.test

```diff
@@ -0,0 +1,66 @@
+nextflow_process {
+
+    name "Test Process DADA2"
+    script "../main.nf"
+    process "DADA2"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "dada2"
+
+    test("Toy test fastq pared reads") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [
+                        file('${moduleDir}/tests/fixtures/test_noambig_1.fastq.gz', checkIfExists: true),
+                        file('${moduleDir}/tests/fixtures/test_noambig_2.fastq.gz', checkIfExists: true) 
+                    ]
+                ]             
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.dada2_out).match() }
```

Comment:
{ assert snapshot(process.out).match() }
to match everything (versions included) to the snapshot